File: URI-py/1115.py
# -*- coding: utf-8 -*-
while True:
    x, y = map(int,input().split())
    if (x==0) or (y==0):
        break
    elif x>0:
        if y>0:
            print("primeiro")
        elif y<0:
            print("quarto")
    elif x<0:
        if y<0:
            print("terceiro")
        elif y>0:
            print("segundo")

File: URI-cpp/1115.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    while(true){
        int x,y;
        cin >> x >> y;
        if (x==0 || y==0){
            break;
        }
        if (x>0){
            if(y>0){
                cout << "primeiro" << endl;
            } else {
                cout << "quarto" << endl;
            }
        } else {
            if(y>0){
                cout << "segundo" << endl;
            } else {
                cout << "terceiro" << endl;
            }
        }
    }
    return 0;
}
